todo.sh

```diff
@@ -489,7 +490,7 @@ OVR_TODOTXT_SORT_COMMAND="$TODOTXT_SORT_COMMAND"
 OVR_TODOTXT_FINAL_FILTER="$TODOTXT_FINAL_FILTER"
 
 # Prevent GREP_OPTIONS from malforming grep's output
-GREP_OPTIONS=""
+export GREP_OPTIONS=""
```

Comment:
We don't really want to override the user's `GREP_OPTIONS`.

Reply:
I don't think export in a child process affects the parent. Example:
```
$ echo $GREP_OPTIONS
--extended-regexp
$ bash -c "export GREP_OPTIONS=''"
$ echo $GREP_OPTIONS
--extended-regexp
```

Reply:
I don't understand this one. In order to affect `todo.sh`, the `GREP_OPTIONS` would need to be exported already in the user's shell. (Or the user does `GREP_OPTIONS=foo todo.sh`, but that's unlikely.) As it's already exported, no need to re-export again from within the script, especially because the script does not need that currently.

Reply:
Primarily this change is intended to fix the shellcheck warning `SC2034: GREP_OPTIONS appears unused. Verify it or export it.`, but addons / actions feature of todo would also benefit from having standardized (i.e. default) grep options. The addon scripts are run as separate processes and do not inherit variables unless they are exported.

Reply:
Okay, thanks for clarifying this. So practically, this doesn't make a difference (as I've previously described), but the shellcheck tool isn't clever enough to detect this, and this is a generic "unused variable" rule, which the `export` satisfies. I'm fine with that.

Reply:
@inkarkat Do we accept this part of revert the `export` part?

Reply:
@karbassi Yes, it's harmless.